### PR TITLE
Add cypress introduction in fr and en

### PR DESCRIPTION
**Pull request by VincentBrule**

### CODE REVIEW

File: posts/2022-06-06-introduction-cypress-fr.adoc [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+ it('should visit', () => {
+   cy.visit('https://blog.lunatech.com/')
+   cy.get(".lb_post").its('length').should('be.gt', 0)
+ })
+})
+----
+
+Le code se lit très facilement. 
+Nous nous attendons à ce qu’il y ait au moins un article sur la page (`gt` correspond à `greater than`). 
+Après avoir sauvegardé, nous voyons le résultat directement dans l’interface graphique :
```
Comment (colineto) [inline posts/2022-06-06-introduction-cypress-fr.adoc:143]:
```suggestion
Après avoir sauvegardé, le résultat apparait dans l’interface graphique :
```

File: posts/2022-06-06-introduction-cypress-fr.adoc [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+
+image::blog-loaded.png[Test vérifiant si les aperçus sont bien affichés, width = 700]
+
+En bas de la partie gauche, vous pouvez voir `expected 36 to be above 0` comme nous voulions.
```
Comment (colineto) [inline posts/2022-06-06-introduction-cypress-fr.adoc:147]:
```suggestion
En bas de la partie gauche, vous pouvez voir `expected 36 to be above 0` qui est bien le résultat attendu.
```

File: posts/2022-06-06-introduction-cypress-fr.adoc [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+
+image::error.png[Affichage d'une erreur dans notre test, width = 500]
+
+Cypress permet vraiment d’avoir un retour rapide et clair afin de déboguer au plus vite notre code !
+
+==== c. Tri par auteur
+Passons à la dernière étape, filtrer par auteur. 
+Un peu plus compliqué, mais _pas impossible_ avec cet outil !
+Première chose à faire, simuler un clic de souris sur un auteur afin d’activer le tri. 
+Nous allons me prendre comme exemple (Vincent Brulé).
```
Comment (colineto) [inline posts/2022-06-06-introduction-cypress-fr.adoc:172]:
```suggestion
Pour cet exemple, l'auteur sera moi-même (Vincent Brulé). 
```

File: posts/2022-06-06-introduction-cypress-fr.adoc [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+Un peu plus compliqué, mais _pas impossible_ avec cet outil !
+Première chose à faire, simuler un clic de souris sur un auteur afin d’activer le tri. 
+Nous allons me prendre comme exemple (Vincent Brulé). 
+Nous devons donc trouver un élément correspondant à mon nom et cliquez dessus. 
+L'élément qui nous intéresse est le suivant :
+
+image::author.png[Élément représentant un auteur, width = 200]
+
+Il y a plusieurs façons de procéder. 
+Pour l’exemple, nous allons chercher un élément avec le texte *Vincent Brulé* sans passer par une classe ou un identifiant.
```
Comment (colineto) [inline posts/2022-06-06-introduction-cypress-fr.adoc:179]: Il y a beaucoup de fois le terme `pour l'exemple`, peut etre ici c'est moins necessaire.
```suggestion
Commençons par essayer de chercher un élément avec le texte *Vincent Brulé* sans passer par une classe ou un identifiant.
```
par exemple

File: posts/2022-06-06-introduction-cypress-fr.adoc [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+image::last-example.png[Résultat de notre exemple, width = 700]
+
+Et si on passe la souris sur la première étape où nous avons sélectionné tous les articles, on voit dans le rendu la page d’accueil avec tous les éléments en bleu : 
+
+image::replay.png[Nous pouvons revoir toutes les étapes une par une, width = 700]
+
+C’est très utile pour déboguer et on peut aussi enregistrer une vidéo de toutes ces actions afin de déboguer plus tard par exemple !
+
+Nous sommes arrivés au bout de ce premier article à propos de Cypress.
+J’espère que je vous ai donné envie d’en savoir plus, car nous allons explorer en détails le fonctionnement de Cypress et les bonnes pratiques dans l’article suivant !
```
Comment (colineto) [inline posts/2022-06-06-introduction-cypress-fr.adoc:216]:
```suggestion
J’espère que je vous ai donné envie d’en savoir plus, car nous allons explorer en détail le fonctionnement de Cypress et les bonnes pratiques dans l’article suivant !
```

File: posts/2022-06-06-introduction-cypress-en.adoc [hunk 6]
```diff
@@ -0,0 +1,216 @@
...
+:lang: en
+:tags: [test, cypress, web, js, ts]
+
+This article is the beginning of a series of three articles dedicated to https://www.cypress.io[Cypress]:
+
+. Introducing Cypress
+. Going Further with Cypress
+. Comparing Cypress and its Competitors
+
+The goal of this first article is to make you want to discover and use *Cypress*.
```
Comment (lukewoodcock) [inline posts/2022-06-06-introduction-cypress-en.adoc:15]: The goal of this first article is to encourage you to discover and use *Cypress*.

Comment (lukewoodcock) [inline posts/2022-06-06-introduction-cypress-en.adoc:15]:
```suggestion
The goal of this first article is to encourage you to discover and use *Cypress*.
```

File: posts/2022-06-06-introduction-cypress-en.adoc [hunk 7]
```diff
@@ -0,0 +1,216 @@
...
+To do so, we will present this tool and put it into practice with a simple example.
+
+== I/ What is Cypress used for?
+
+Tests are and always have been an important point in a project. 
+There are many types of testing such as unit testing and integration testing. 
+Cypress focuses on end-to-end testing (E2E).
+The E2E tests allow us to validate the user experience through predefined scenarios. 
+These scenarios will describe a group of actions that an average user will perform on our site. 
+This kind of test does not care about the internal implementation of our product, but will adopt the point of view of a user to verify the general behavior of our product.
```
Comment (lukewoodcock) [inline posts/2022-06-06-introduction-cypress-en.adoc:25]:
```ssuggestion
These tests do not care about the internal implementation of our product, but will adopt the point of view of a user to verify the general behaviour of our product.
```
It's perhaps a little pedantry, but in the previous sentences you refer to test (plural), then suddenly you switch to the singular. No biggy if you leave it, just change the typo "behavior" to "behaviour"

Comment (VincentBrule) [reply posts/2022-06-06-introduction-cypress-en.adoc:25]: That's a good point, thanks 👍

File: posts/2022-06-06-introduction-cypress-en.adoc [hunk 8]
```diff
@@ -0,0 +1,217 @@
...
+. Introducing Cypress
+. Going Further with Cypress
+. Comparing Cypress and its Competitors
+
+The goal of this first article is to encourage you to discover and use *Cypress*.
+To do so, we will present this tool and put it into practice with a simple example.
+
+== I/ What is Cypress used for?
+
+Tests are and always have been an important point in a project.
```
Comment (thinkmorestupidless) [inline posts/2022-06-06-introduction-cypress-en.adoc:20]:
```suggestion
Tests are and always have been important in a project. 
```

File: posts/2022-06-06-introduction-cypress-en.adoc [hunk 9]
```diff
@@ -0,0 +1,217 @@
...
+To save you time and not type the full path to the Cypress executable, I suggest you put this script in your `package.json` :
+[source, Shell]
+----
+  "scripts": {
+    "cypress:open": "cypress open"
+  }
+----
+
+[quote, How to organize the code?]
+In this first article about Cypress, we will organize the code very simply, but in the following article we will explore good practices to have a clean and maintainable code over time.
```
Comment (thinkmorestupidless) [inline posts/2022-06-06-introduction-cypress-en.adoc:65]:
```suggestion
In this first article about Cypress, we will organize the code very simply, but in the following article we will explore good practices to ensure clean and maintainable code over time.
```

File: posts/2022-06-06-introduction-cypress-en.adoc [hunk 10]
```diff
@@ -0,0 +1,217 @@
...
+In our case, we will use _Chrome_.
+Cypress will then create a base project to start your tests.
+If all went well, you should have this file structure:
+
+image::fichiers-cypress.png[Files created by Cypress, width = 300]
+
+Now you have everything in place to move on to the practical case. 
+Don't close the GUI so you can see your changes in real time!
+
+=== 2. Realization of the practical case
```
Comment (thinkmorestupidless) [inline posts/2022-06-06-introduction-cypress-en.adoc:92]:
```suggestion
=== 2. Implementation
```
This is the simplest way to express what i _think_ you mean here - if you'd prefer something less succinct then, perhaps:

```suggestion
=== 2. Building out the example
```

**Review by thinkmorestupidless — APPROVED** [review]
Just a couple of pretty minor things, Vincent... and Cypress looks really interesting, thanks!

**Pull request merged**